.github/workflows/ci.yml

```diff
@@ -219,7 +219,9 @@ jobs:
               uses: actions/checkout@v1
 
             - name: Install the dependencies
-              run: composer install --ignore-platform-reqs --no-interaction --no-progress
+              run: |
+                  composer install --ignore-platform-req=php --no-interaction --no-progress
+                  composer require "symfony/http-foundation:5.3.*@dev" --ignore-platform-req=php --no-interaction --no-progress
```

Comment:
Can you explain this change? Or add a comment for anyone to understand?

Reply:
Then we should add a TODO to remove that again once it's fixed, no? I just want to make sure the test setup is fixed afterwards as we're probably testing a wrong combination otherwise :)

Reply:
```suggestion
                  # TODO: Remove the next line once symfony 5.3.7 was released
                  composer require "symfony/http-foundation:5.3.*@dev" --ignore-platform-req=php --no-interaction --no-progress
```
Like so?

Reply:
Yessa :)